article/_posts/2022-04-14-MultiModalVersatileNetworks.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: review
+title: "Self-Supervised MultiModal Versatile Networks"
+tags: deep-learning representation-learning self-supervised multi-task
+author: "Nathan Painchaud"
+cite:
+    authors: "Jean-Baptiste Alayrac, Adria Recasens, Rosalia Schneider, Relja Arandjelović, Jason Ramapuram, Jeffrey De Fauw, Lucas Smaira, Sander Dieleman, Andrew Zisserman"
+    title:   "Self-Supervised MultiModal Versatile Networks"
+    venue:   "NeurIPS 2020"
+pdf: "https://proceedings.neurips.cc/paper/2020/file/0060ef47b12160b9198302ebdb144dcf-Paper.pdf"
+---
+
+
+# Highlights
+- Introduce the notion of a MultiModal Versatile (MMV) network, that can ingest multiple modalities and outputs
+representations useful for downstream tasks in multiple modalities;
```

Comment:
Looks a redundancy: I'd to say that the representations would be useful for "(...) representations useful for multiple downstream tasks." If multiple modalities are input, I'd say that downstream tasks are implicitly useful for "multiple modalities".

Reply:
Good point! I think I failed to mention that the representations are shared, and that's why I need to insist on the use for multiple modalities. I rephrased this to now simply say: "ingest multiple modalities and outputs _common_ representations useful for downstream tasks".

Reply:
Addressed in c9d2a78.